acos_client/v30/axapi_http.py

```diff
@@ -74,9 +74,12 @@ def request(self, method, api_url, params={}, headers=None,
             hdrs.update(headers)
 
         if params:
-            extra_params = kwargs.get('axapi_args', {})
+            # FIXME - re-enable kwargs merge at some point; dict keys
```

Comment:
Do you want to adopt the `FIXME(dougwig)` here? 

I don't care, just thought I'd remind.